client/src/entrypoints/admin/core.test.js

```diff
@@ -25,8 +23,4 @@ describe('core', () => {
       expect.arrayContaining(['Application', 'Controller']),
     );
   });
-
-  it('DOMContentLoaded', () => {
```

Comment:
Note: This test should have been removed earlier, we stopped using DOMContentLoaded recently but it started failing when our other afterLoad/DomReady functions were removed.